draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3456,808 +3548,1280 @@ ObjectLists MAY combine regular objects and secondary ObjectLists in the same ob
+    </t>
+
+
+    <figure><artwork><![CDATA[
+REQUEST:
+
+GET /cit HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
 
 RESPONSE:
 
-  HTTP/1.1 200 OK
-  Content-Length: 341
-  Expires: Wed, 04 May 2016 08:49:11 GMT
-  Server: example-server/0.1
-  ETag: "-936094426920308378"
-  Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:11 GMT
-  Content-Type: application/cdni; ptype=ci-trigger-collection
+HTTP/1.1 200 OK
+Content-Length: 341
+Expires: Sun, 27 Oct 2024 08:49:18 GMT
+Server: example-server/0.1
+ETag: "936094426920308378"
+Last-Modified: Sun, 27 Oct 2024 08:40:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:18 GMT
+Content-Type: application/cdni; ptype=ci-trigger-collection.v2
 
 {
   "cdn-id": "AS64496:0",
-  "coll-status": [
+  "coll-state": [
     {
-      "status"    : "pending",
-      "collection": "/triggers/pending"
+      "state": "pending",
+      "uri": "/cit/state/pending"
     },
     {
-      "status"    : "active",
-      "collection": "/triggers/active"
+      "state": "active",
+      "uri": "/cit/state/active"
     },
     {
-      "status"    : "complete",
-      "collection": "/triggers/complete"
+      "state": "complete",
+      "uri": "/cit/state/complete"
     },
     {
-      "status"    : "processed",
-      "collection": "/triggers/processed"
+      "state": "processed",
+      "uri": "/cit/state/processed"
     },
     {
-      "status"    : "failed",
-      "collection": "/triggers/failed"
+      "state": "failed",
+      "uri": "/cit/state/failed"
     },
     {
-      "status"    : "cancelling",
-      "collection": "/triggers/cancelling"
+      "state": "cancelling",
+      "uri": "/cit/state/cancelling"
     },
     {
-      "status"    : "cancelled",
-      "collection": "/triggers/cancelled"
+      "state": "cancelled",
+      "uri": "/cit/state/cancelled"
     }
   ],
   "coll-label": [
     {
-      "label"     : "1b1bad0c", 
-      "collection": "/triggers/1b1bad0c"
-    },
-    {
-      "label"     : "fafa9a97",
-      "collection": "/triggers/fafa9a97"
+      "label": "type=video", 
+      "uri": "/cit/labels/type=video"
     }
   ],
   "staleresourcetime": 86400,
   "triggers": [
-    "https://dcdn.example.com/triggers/0",
-    "https://dcdn.example.com/triggers/1"
+    "https://dcdn.example/cit/991b9fb9-d0be-4d05-be06-64c0e5c5a5f9",
+    "https://dcdn.example/cit/86633e6e-d2da-4185-a285-b3d087a5d711"
   ]
 }
 ]]></artwork>
-   </figure>
-   </section>
+    </figure>
+    </section>
 
-   <section title="Filtered Collections of Triggers" anchor="filtered-collections"><t>
-   The filtered collections are also available to uCDN. Before the
-   dCDN starts processing the two triggers shown above,
-   both will appear in the collection of pending triggers. For example:</t>
+    <section title="Filtering of Trigger Collection" anchor="filtering-trigger--collection">
+    <t>
+    Before the dCDN starts processing the two triggers shown above,
```

Comment:
There are now 4 trigger examples, although 1 is canceled and deleted--and they aren't the same triggers in the collection of all triggers.